cmd/gossamer/config.go

```diff
@@ -26,24 +24,22 @@ import (
 
 var (
 	// DefaultCfg is the default configuration for the node.
-	DefaultCfg                = dot.GssmrConfig
-	defaultGssmrConfigPath    = "./chain/gssmr/config.toml"
-	defaultKusamaConfigPath   = "./chain/kusama/config.toml"
-	defaultPolkadotConfigPath = "./chain/polkadot/config.toml"
-	defaultDevConfigPath      = "./chain/dev/config.toml"
-
-	gossamerName = "gssmr"
-	kusamaName   = "kusama"
-	polkadotName = "polkadot"
-	devName      = "dev"
+	DefaultCfg                  = dot.WestendDevConfig
+	defaultKusamaConfigPath     = "./chain/kusama/config.toml"
+	defaultPolkadotConfigPath   = "./chain/polkadot/config.toml"
+	defaultWestendDevConfigPath = "./chain/westen-dev/config.toml"
```

Comment:
typo this is meant to be

```suggestion
	defaultWestendDevConfigPath = "./chain/westend-dev/config.toml"
```